test/Messages.t.sol

```diff
@@ -91,41 +91,41 @@ contract MessagesTest is Test {
 
     function testUpdateMemberEncoding() public {
         assertEq(
-            ConnectorMessages.formatUpdateMember(5, toBytes16(fromHex("010000000000000003")), 0x225ef95fa90f4F7938A5b34234d14768cB4263dd, 1657870537), 
-            fromHex("04000000000000000500000000000000000000000000000009225ef95fa90f4f7938a5b34234d14768cb4263dd0000000000000000000000000000000000000000000000000000000062d118c9")
-            );
+            ConnectorMessages.formatUpdateMember(2, bytes16(hex"811acd5b3f17c06841c7e41e9e04cb1b"), 0x1231231231231231231231231231231231231231, 1706260138),
```

Comment:
An interesting and frustration finding was that `toBytes16(fromHex("811acd5b3f17c06841c7e41e9e04cb1b"))` was retuning me an empty `bytes16` 🤷‍♂️

Reply:
Odd... does `fromHex()` expect a prepended `0x` maybe?

Reply:
Doesn't really look like it based on how we use it elsewhere actually.